services/storage/src/simcore_service_storage/datcore_adapter/datcore_adapter_settings.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional
+
+from models_library.basic_types import PortInt, VersionTag
+from pydantic import AnyHttpUrl, Field, validator
+from settings_library.base import BaseCustomSettings
+
+
+class DatcoreAdapterSettings(BaseCustomSettings):
+    ENABLED: bool = True
```

Comment:
Here again, field names should be same as env var names, 

```python

DATCORE_ADAPTER_ENABLED: bool = True
...
```


TIP: replace with the following regex
Find:  ``(\w+):\s``
Replace: ``DATCORE_ADAPTER_\U$1: ``. (notice the last character is a space!)